tool/tctl/common/resource_command.go

```diff
@@ -110,6 +110,8 @@ func (rc *ResourceCommand) Initialize(app *kingpin.Application, config *service.
 		types.KindToken:                   rc.createToken,
 		types.KindInstaller:               rc.createInstaller,
 		types.KindNode:                    rc.createNode,
+		types.KindOIDCConnector:           rc.createOIDCConnector,
+		types.KindSAMLConnector:           rc.createSAMLConnector,
```

Comment:
Are these changes part of the effort to get rid of enterprise-specific tctl?

Reply:
Kind of. They're necessary because we're leveraging `tctl`'s `ResourceCommand` to get the existing resource.

If we keep today's approach of having a `ResourceCommand` and a ResourceCommandE`, then we would have to create an `EditCommand` and an `EditCommandE`.

Since we're trying to get rid of enterprise tctl, I consolidated on a single resource command. This means you can now create OIDC and SAML connectors with OSS tctl or enterprise tctl, so long as the auth server is enterprise.